messaging/src/test/java/org/axonframework/eventhandling/deadletter/DeadLetteringEventIntegrationTest.java

```diff
@@ -558,6 +563,26 @@ void processedDeadLetterIsResolvedAsParameterToEventHandlers() {
         assertEquals(3, eventHandlingComponent.resolvedDeadLetterParameterCount(aggregateId));
     }
 
+    @Test
+    void deadLetterEventProcessingTaskIsUsingInterceptor() {
```

Comment:
I would append the same comment here as I placed on the `DeadLetteredEventProcessingTask`.

Reply:
added in [172a5b3](https://github.com/AxonFramework/AxonFramework/pull/2661/commits/172a5b395c9c80f283a8893f1295b203e2a8b399)